sweep/tx_input_set.go

```diff
@@ -83,14 +83,18 @@ type txInputSet struct {
 	wallet Wallet
 }
 
+func dustLimit(relayFee chainfee.SatPerKWeight) btcutil.Amount {
+	return txrules.GetDustThreshold(
```

Comment:
this will be wrong for litecoin..

Reply:
oops. Added to the tracking issue: https://github.com/lightningnetwork/lnd/issues/3946